less/mixins.less

```diff
@@ -1 +1,13 @@
-// Mixins in here
\ No newline at end of file
+// Mixins in here// Mixins in here
+.btn(@color: white, @bgcolor: darkcyan) {
+  font-size: 1.7rem;
+  color: @color;
+  background-color: @bgcolor;
+  width: 80%;
+  height: 40px;
+  border-radius: 13px;
+  text-align: center;
+  padding-top: 12px;
+}
+
```

Comment:
nice work with:
- matching the desktop design
- using a parametric mixin for the buttons